app/src/main/java/com/google/android/stardroid/activities/dialogs/TimeTravelDialog.java

```diff
@@ -185,8 +185,8 @@ private void setDate(Date date) {
 
   private void updateDisplay() {
     Date date = calendar.getTime();
-    dateTimeReadout.setText(String.format(parentActivity.getString(R.string.now_visiting,
-                                                                   dateFormat.format(date))));
+    dateTimeReadout.setText(String.format(parentActivity.getString(R.string.now_visiting)+
```

Comment:
When does this crash?  What should be happening here is the string gets interpolated. It's not guaranteed that appending the date to the string is the correct way to do it in every language, so the string needs to have a place holder that the translators can accommodate.

Reply:
@jaydeetay Have you had a look at #277  . Why did you close it. It needed some refactoring.

Reply:
@jaydeetay I shall put a placeholder in the string for the date. Will that be cool?